nixos/modules/services/misc/mpdscribble.nix

```diff
@@ -0,0 +1,292 @@
+            readable /nix/store.
+          ";
+        };
+
+        journal = mkOption {
+          default = "/tmp/lastfm-journal.mpdscribble";
+          type = types.path;
+          description = "
+            Journal file for the lastfm backend.
+            Should be changed from the default.
+          ";
+        };
+
+      };
+
+      librefm = {
+        enable = mkOption {
+          default = false;
+          type = types.bool;
+          description = "
+            Whether to enable the mpdscribble daemon for librefm.
+          ";
+        };
+
+        url  = mkOption {
+          default = false;
+          type = types.str;
+          description = "
+            Whether to enable the mpdscribble daemon for librefm.
+          ";
+        };
+
+        username = mkOption {
+          default = "";
+          type = types.str;
+          description = "
+            Librefm username.
+          ";
+        };
+
+        password = mkOption {
+          default = false;
+          type = types.str;
+          description = "
+            Librefm Password. Attention: This will be stored in the globally
+            readable /nix/store.
+          ";
+        };
+
+        journal = mkOption {
+          default = "/tmp/librefm-journal.mpdscribble";
+          type = types.path;
+          description = "
+            Journal file for the librefm backend.
+            Should be changed from the default.
+          ";
+        };
+
+      };
+
+      jamendo = {
+        enable = mkOption {
+          default = false;
+          type = types.bool;
+          description = "
+            Whether to enable the mpdscribble daemon for jamendo.
+          ";
+        };
+
+        url  = mkOption {
+          default = false;
+          type = types.str;
+          description = "
+            Whether to enable the mpdscribble daemon for jamendo.
+          ";
+        };
+
+        username = mkOption {
+          default = "";
+          type = types.str;
+          description = "
+            Jamendo username.
+          ";
+        };
+
+        password = mkOption {
+          default = false;
+          type = types.str;
+          description = "
+            Jamendo Password. Attention: This will be stored in the globally
+            readable /nix/store.
+          ";
+        };
+
+        journal = mkOption {
+          default = "/tmp/jamendo-journal.mpdscribble";
+          type = types.path;
+          description = "
+            Journal file for the jamendo backend.
+            Should be changed from the default.
+          ";
+        };
+
+      };
+
+    };
+
+  };
+
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+      systemd.services.mpdscribble = {
+        after                   = [ "network.target" ];
+        description             = "mpdscribble mpd scrobble client";
+        wantedBy                = "multi-user.target";
+        path                    = [ pkgs.mpdscribble ];
```

Comment:
This line should be not necessary because you start mpdscribble with the full path.